api/definitions/definitions.proto

```diff
@@ -0,0 +1,895 @@
+syntax = "proto3";
```

Comment:
looks like we have two copies of this file?

should we put it under `resource/*` ?

Reply:
Is it okay to put it there? I mean we usually store it in `/api`. `make docs` expect `.proto` files to be there.

Reply:
I mean under `/api/resource/*`, as we already have it

Reply:
Done!